ros_buildfarm/status_page.py

```diff
@@ -1047,9 +1047,15 @@ def write_summary(summary_filename, ordered_pkgs, repos_data):
         pkg_d['maintainers'] = []
         for maintainer in pkg.maintainers:
             maintainer_dict = {'email': maintainer.email}
-            try:
-                maintainer_dict['name'] = str(maintainer.name)
-            except:
+            # In order to write the simplest data to the yaml, we try to encode the name to ascii.
+            ascii_name = maintainer.name.decode('utf-8').encode('ascii', 'ignore')
+            if maintainer.name == ascii_name:
+                # If it is the same, we write the ascii name to the dict
+                # Otherwise it will appear in the file as
+                # name: !!python/unicode 'David V. Lu!!'
+                maintainer_dict['name'] = ascii_name
+            else:
+                # Otherwise, we write in all its unicode glory
```

Comment:
Instead of this approach I would suggest to "suppress/avoid" the `!!python/unicode` when dumping the data (e.g. using a custom dumper class).

But why do you actually care if `!!python/unicode` is in the generated yaml? Are there any cases where you can't read that file?

Reply:
* Readability of the file
 * Portability to other languages
 * File size